components/event-publisher-proxy/pkg/metrics/collector.go

```diff
@@ -75,11 +94,16 @@ func (c *Collector) RecordError() {
 }
 
 // RecordLatency records a latency metric
-func (c *Collector) RecordLatency(duration time.Duration) {
-	c.latency.WithLabelValues().Observe(duration.Seconds())
+func (c *Collector) RecordLatency(duration time.Duration, statusCode int, destSvc string) {
```

Comment:
(nits) lets also change to argument to `responseCode` to be consistent.

Reply:
I get your point but here we use `statusCode` as well https://github.com/kyma-project/kyma/blob/3b7ca96d712d155ea1bb887bebf05c82c726050e/components/eventing-controller/pkg/handlers/metrics/collector.go#L63
My suggestion would be to also change it in `ec` to `responseCode` to be really consitent.
Let me know what you think.

Reply:
okay lets keep statusCode for now.